Lux/src/lux_web.cpp

```diff
@@ -1423,6 +1438,246 @@ std::string ultra_get_camera_stats() {
     return stats.dump();
 }
 
+// Audio context management functions
+void update_audio_context(float volume, float bass, float mid, float high, bool beat, float time) {
```

Comment:
I would like more fine-grained control over audio input. For instance, I'd like to be able to specify frequency and bandwidth for each function. This means that when scene is loaded we may need to add specific SHO or FFT components for the specified functions, and change them if audio input parameters change.